client/components/command-palette/index.tsx

```diff
@@ -109,14 +104,14 @@ export function CommandMenuGroup( {
 							<LabelWrapper>
 								<Label>
 									<TextHighlight text={ command.label } highlight={ search } />
+									{ command.siteFunctions && '...' }
```

Comment:
Should we actually add the ellipsis to the label so it can be properly translated?

Also, we should use a true ellipsis instead of three periods.

Reply:
@katinthehatsite Sorry, to clarify, I meant something like this:

```diff
diff --git a/client/sites-dashboard/components/wpcom-smp-commands.tsx b/client/sites-dashboard/components/wpcom-smp-commands.tsx
index 3cffd7a8080..c93f22492e0 100644
--- a/client/sites-dashboard/components/wpcom-smp-commands.tsx
+++ b/client/sites-dashboard/components/wpcom-smp-commands.tsx
@@ -175,7 +175,7 @@ export const useCommandsArrayWpcom = ( {
 		},
 		{
 			name: 'openPHPmyAdmin',
-			label: __( 'Open database in phpMyAdmin' ),
+			label: __( 'Open database in phpMyAdmin…' ),
 			context: [ '/sites' ],
 			callback: setStateCallback( 'openPHPmyAdmin' ),
 			siteFunctions: {

```

Reply:
Oh I see! Hmmm I don't know if the ellipsis need to be translated though as they are not for the meaning of the sentence but for the action indication? If we translate it, it would be seen as an incomplete sentence, no? 

I am also not sure I like them manually added like so in each command 🤔 

@sejas - what is your opinion?

Reply:
Sounds good, I updated the character.